client/functions/basebuilding/fn_abort_building.sqf

```diff
@@ -39,4 +39,9 @@ if (!isNil "para_l_placing_keyrotate_handler") then {
 	para_l_placing_keyrotate_handler = nil;
 };
 
+if (!(isNil "para_l_placing_scroll_handler")) then {
```

Comment:
What dis?

Reply:
This was something I was working on my own, did not mean to put this in the pull request. Still getting used to GitHub lol

Reply:
all good. what was it out of interest?

Reply:
Was inspired by Legend's 'r' rotate key. I tried doing essentially the same thing but with the mouse wheel. This is the handler that stops the mouse rotation.